packages/conveyer/src/Conveyer.ts

```diff
@@ -437,16 +447,41 @@ class Conveyer extends Component<ConveyerEvents> {
         this._isAnimationScroll = !this._isWheelScroll && !isHold;
         isDrag = true;
         const scroll = e.delta.scroll;
-
         if (options.horizontal) {
           scrollAreaElement.scrollLeft -= scroll;
         } else {
           scrollAreaElement.scrollTop -= scroll;
         }
+        if (!e.isTrusted && animateParam?.duration) {
+          const easing = this._axes!.options.easing!;
+          const diffTime = new Date().getTime() - animateParam.startTime;
+          const ratio = diffTime / animateParam.duration;
+          const delta = animateParam.destPos - animateParam.depaPos;
+          const scrollPos = options.horizontal ? scrollAreaElement.scrollLeft : scrollAreaElement.scrollTop;
+          const expectedPos = animateParam.depaPos + easing(ratio) * delta;
+          const diffPos = expectedPos - scrollPos;
```

Comment:
Isn't expectedPos the sum of `delta.scroll`?

Reply:
Using the sum of `delta.scroll` would save a lot of unnecessary math, I updated the code applying this review.